models/action.js

```diff
@@ -15,6 +15,7 @@ const ActionSchema = new Schema({
   user_id: String,
   field: String, // Used when an action references a particular field of an object. (e.g. a flag on a username or bio)
   detail: String, // Describes the reason for an action (e.g. 'Username is offensive')
+  description: String, // Describes the reason for the action in more detail (e.g. A description of why the comment is offensive.)
```

Comment:
Also, +1 to using `metadata.notes` instead of `metadata.comment`. Using comment here is likely to confuse devs thinking that comment may refer to a comment's id or something.

`notes` is a good generic name for fields populated by user input to embellish something else.